anime_search_engine.py

```diff
@@ -0,0 +1,95 @@
+from whoosh.index import create_in, open_dir, exists_in
+from whoosh.fields import *
+from whoosh.analysis import StemmingAnalyzer
+from whoosh.qparser import QueryParser, OrGroup, AndGroup
+from whoosh import scoring
+
+from bs4 import BeautifulSoup
+
+import os, pickle
+
+class SearchEngine(object):
+
+	def __init__(self):
+		self.schema = Schema(title=TEXT(stored=True), url = ID(stored=True), content=TEXT(analyzer=StemmingAnalyzer()))
+		self.ix = self.__get_indexer("./indexdir")
```

Comment:
It would be a good idea to have the filenames be optional parameters when initializing the SearchEngine instance.